2023/10/27/main.go

```diff
@@ -0,0 +1,18 @@
+// Daily Joel 2023-10-27
+//
+// Structs.
+package main
+
+import "fmt"
+
+type Person struct {
+	name *string
+}
+
+func main() {
+	n := "Joel"
+	p := Person{
+		name: &n,
+	}
+	fmt.Println(*p.name)
+}
```

Comment:
Yes, not sure where I was going with this tbh since it doesn't really harvest the value from having the `name` field as a pointer. I think I just wanted to showcase that struct fields can be pointers, even though the struct itself might not be accessed through a pointer

Reply:
You are correct that this would yield the same result if `name` was not a pointer